### PR TITLE
`help`: add available commands list in connect

### PR DESCRIPTION
**Pull request by sssciel**
Add lines with usage connect  commands in
`tt help connect`

### CODE REVIEW

**Comment by sssciel** [comment]
image

File: cli/cmd/connect.go [hunk 1]
```diff
@@ -53,7 +53,8 @@ func NewConnectCmd() *cobra.Command {
 			"  COMMAND | tt connect (<APP_NAME> | <APP_NAME:INSTANCE_NAME> | <URI>)" +
 			" [flags]\n" +
 			"  COMMAND | tt connect (<APP_NAME> | <APP_NAME:INSTANCE_NAME> | <URI>)" +
-			" [flags] [-f-] [-- ARGS]\n\n" +
+			" [flags] [-f-] [-- ARGS]\n" +
+			"  \\shortcuts - to get full list of available shortcuts in go prompt\n\n" +
```
Comment (0x501D) [inline cli/cmd/connect.go:57]: get the full list of available hotkeys for go prompt

**Comment by sssciel** [comment]
image

File: cli/cmd/connect.go (hunk 1 above)
Comment (sssciel) [reply cli/cmd/connect.go]: fixed

**Review by 0x501D — APPROVED** [review]

**Review by askalt — COMMENTED** [review]
Hi! Thanks for the patch! Could you please also add other supported commands? Like `\set language`.

**Review by askalt — APPROVED** [review]

**Pull request merged**